tools/cldr-code/src/main/java/org/unicode/cldr/util/GrammarInfo.java

```diff
@@ -75,8 +75,15 @@ public String getShortName() {
         public CharSequence getSymbol() {
             return symbol;
         }
-        public String getDefault(Collection<String> values) {
-            return this == grammaticalGender && values != null && !values.contains("neuter") ? "masculine" : defaultValue;
+        /**
+         * Gets the default value. The parameter only needs to be set for grammaticalGender
+         */
+        public String getDefault(Collection<String> featureValuesFromGrammaticalInfo) {
+            return this == grammaticalGender
+                && featureValuesFromGrammaticalInfo != null
+                    && !featureValuesFromGrammaticalInfo.contains("neuter")
+                    ? "masculine"
+                        : defaultValue;
```

Comment:
Possibility for future improvement: define constants for "neuter", etc. If one of them were misspelled, for example, the compiler would catch the problem. Constants might facilitate updating the code when additional genders are added, etc.

Reply:
such could be derived from the DTD +supplemental data

Reply:
Those exist already; good point about using them.